environment.yml

```diff
@@ -0,0 +1,37 @@
+name: nf-core-coproid-1.0dev
+channels:
+  - conda-forge
+  - bioconda
+  - maxibor
+  - etetoolkit
+  - defaults
+dependencies:
+  - bioconda::adapterremoval=2.2.2
+  - bioconda::bcftools=1.9
+  - bioconda::bedtools=2.27.1
+  - bioconda::blast=2.5.0
+  - anaconda::bokeh=1.0.4
+  - bioconda::bowtie2=2.3.5
+  - bioconda::damageprofiler=0.4.6
+  - bioconda::ete3=3.1.1
+  - bioconda::fastqc=0.11.8
+  - bioconda::htslib=1.9
+  - anaconda::jupyter=1.0.0
+  - bioconda::kraken2=2.0.7_beta
+  - conda-forge::notebook=5.7.5
+  - anaconda::nbconvert=5.4.1
+  - bioconda::nextflow=19.01.0
+  - bioconda::pmdtools=0.60 
+  - bioconda::pysam=0.15.2
+  - maxibor::samtools=1.9
```

Comment:
```suggestion
  - bioconda::samtools=1.9
```

Reply:
I'd even suggest moving your stuff (as its bio stuff) to bioconda. Since you already have a recipe, it should be fairly easy to do so 👍

Reply:
Agree, I forgot to switch samtools back to bioconda (had to build it on my channel when there was the openssl issue on bioconda).
For sourcepredict, I want to put on Bioconda when I submit it to [JOSS](https://github.com/openjournals/joss) (I'd like to put it on pypi as well), so it should stay on my channel for the moment.

Reply:
Samtools from bioconda with #7